tools/decompress_baserom.py

```diff
@@ -235,6 +251,12 @@ def main():
     file_content = per_version_fixes(file_content, version)
 
     dmadata = read_dmadata(file_content, file_table_offset)
+    if len(segment_names) != len(dmadata):
+        print(
+            f"Error: segment names and dmadata length mismatch ({len(segment_names)} != {len(dmadata)})"
```

Comment:
```suggestion
            f"Error: segment names and dmadata length mismatch ({len(segment_names)} != {len(dmadata)})", file=sys.stderr
```
Along with including sys
For these error messages we should probably be writing to stderr.